include/z64pause.h

```diff
@@ -87,14 +87,14 @@ typedef enum PauseState {
 typedef enum PauseMainState {
     /* 0 */ PAUSE_MAIN_STATE_IDLE,
     /* 1 */ PAUSE_MAIN_STATE_SWITCHING_PAGE,
-    /* 2 */ PAUSE_MAIN_STATE_2,
+    /* 2 */ PAUSE_MAIN_STATE_SONG_PLAYBACK,
     /* 3 */ PAUSE_MAIN_STATE_3,
-    /* 4 */ PAUSE_MAIN_STATE_4,
-    /* 5 */ PAUSE_MAIN_STATE_5,
-    /* 6 */ PAUSE_MAIN_STATE_6,
+    /* 4 */ PAUSE_MAIN_STATE_SONG_PROMPT_INIT,
+    /* 5 */ PAUSE_MAIN_STATE_SONG_PROMPT,
+    /* 6 */ PAUSE_MAIN_STATE_SONG_PROMPT_DONE,
```

Comment:
is "prompt" the demo, or the state where its waiting for the player to play it? similar question for `PAUSE_MAIN_STATE_SONG_PLAYBACK_START` I guess, does this mean the player plays it back or the pause menu does?

Disclaimer I read 0 code below, just giving initial thoughts

Reply:
playback is the game playing, prompt is the player playing

this was hard to come up with initially, I remember we spent some time with engineer trying to find better names for the audio ocarina docs and didn't really find anything better

Reply:
maybe some comments next to state names to make clear between "song demo" and "inputting song(?)" would help

Reply:
(not relevant to this PR but calling this "song prompt" also makes me wish we called the save/continue prompts something more specific than "kaleido prompt")